server/graphql/v1/mutations/collectives.js

```diff
@@ -381,7 +381,16 @@ export function editCollective(_, args, req) {
         return collective.updateCurrency(newCollectiveData.currency, req.remoteUser);
       }
     })
-    .then(() => collective.update(omit(newCollectiveData, ['HostCollectiveId', 'hostFeePercent', 'currency']))) // we omit those attributes that have already been updated above
+    .then(() => {
+      // Set private instructions
+      const privateInstructions = args.collective.data?.privateInstructions;
```

Comment:
`collective.data` is deprecated on the `input` too: https://github.com/opencollective/opencollective-api/blob/1d321f312479e98cfd99dcfd34d20223fe31fdeb/server/graphql/v1/inputTypes.js#L171

We need to add a new `privateInstructions` field on `CollectiveInputType` as well.

Reply:
Updated 👍